MonoGame.Framework/Graphics/ModelBone.cs

```diff
@@ -22,26 +25,30 @@ public sealed class ModelBone
 			}
 		}
 
-		// Summary:
-		//     Gets a collection of bones that are children of this bone.
-		public ModelBoneCollection Children { get; private set; }
-		//
-		// Summary:
-		//     Gets the index of this bone in the Bones collection.
+        /// <summary>
+        /// Gets a collection of bones that are children of this bone.
+        /// </summary>
+        public ModelBoneCollection Children { get; private set; }
+
+        /// <summary>
+        /// Gets the index of this bone in the <see cref="Model.Bones">Model.Bones</see> collection.
+        /// </summary>
 		public int Index { get; set; }
-		//
-		// Summary:
-		//     Gets the name of this bone.
+
+        /// <summary>
+        /// Gets the name of this bone.
+        /// </summary>
 		public string Name { get; set; }
-		//
-		// Summary:
-		//     Gets the parent of this bone.
+
+        /// <summary>
+        /// Gets the parent of this bone.
+        /// </summary>
 		public ModelBone Parent { get; set; }
-		//
-		// Summary:
-		//     Gets or sets the matrix used to transform this bone relative to its parent
-		//     bone.
+
 		internal Matrix transform;
+        /// <summary>
+        /// Gets or sets the matrix used to transform this bone relative to its parent bone.
```

Comment:
VERY GOOD!  explains why this parameter is important and its use, more of this please!